frontend/components/ResourceCounts.tsx

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
+import { Badge, Button, Stack } from '@mantine/core';
+import Link from 'next/link';
+
+/**
+ * Component which retrieves all resources and their counts and translates them into buttons
+ * @returns array of JSX Buttons
+ */
+const ResourceCounts = () => {
+  // set initial state to no resources
+  const [resources, setResources] = useState<Record<string, number>>({
+    Measure: 0,
+    Library: 0
+  });
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
I think that if another error message were to be used in the future, you'd probably want to have it show up similarly. Since this is handling state for rendering components, I would say that any error message could be set in this state as long as you want it to show up out of this component.